src/main/java/ui/components/PanelMenuBar.java

```diff
@@ -87,27 +88,33 @@ private HBox createNameArea() {
                 activateInplaceRename();
             }
         });
-        Tooltip.install(nameArea, new Tooltip("Edit the name of this panel"));
+        Tooltip.install(nameArea, new Tooltip("Double click to edit the name of this panel"));
 
+        nameArea.getChildren().add(nameBox);
+        nameArea.setMinWidth(NAME_AREA_WIDTH);
+        nameArea.setMaxWidth(NAME_AREA_WIDTH);
+        nameArea.setPadding(new Insets(0, 10, 0, 5));    // Looks nicer with some padding
```

Comment:
That's true, but I think the intent of that line is clear. :stuck_out_tongue: No need for the comment.